content/ja/docs/contributing/pull-requests.md

```diff
@@ -0,0 +1,467 @@
+class 1,2,3,4,5,6,7,8 grey
+class first,second white
+```
+
+_図 4. フォークから、PR を公開する手順_
+[opentelemetry.io](https://github.com/open-telemetry/opentelemetry.io).
+
+1. Web ブラウザで [`opentelemetry.io`](https://github.com/open-telemetry/opentelemetry.io) リポジトリにアクセスしてください。
+1. **New Pull Request** を作成してください。
+1. **compare across forks** を選択してください。
+1. **head repository** ドロップダウンメニューから、あなたのフォークを選択してください。
+1. ドロップダウンメニューの **compare** から、あなたのブランチを選択してください。
+1. **Create Pull Request** を選択してください。
+1. プルリクエストの説明を追加してください。
+
+   - **タイトル** (50 文字未満): 変更の意図を要約してください。
+   - **説明**: 変更を詳細に記述してください。
+
+     - GitHub イシューに関連する場合、`Fixes #12345` や `Closes #12345` を説明に記述することで、PR のマージしたあとに GitHub の自動化が該当イシューをクローズします。 他の関連する PR がある場合もリンクしてください。
+     - 特定のことに対してアドバイスを求める場合、説明にレビュアーが考えて欲しい質問を含めてください。
+
+1. **Create pull request** ボタンを選択してください。
+
+プルリクエストは、[Pull requests](https://github.com/open-telemetry/opentelemetry.io/pulls) で確認できます。
+
+PR を公開した後に、自動テストの実行と [Netlify](https://www.netlify.com/) を使用したプレビューのデプロイを試みます。
+
+- Netlify ビルドが失敗した場合、詳細な情報のために **Details** を選択してください。
+- Netify のビルドが成功した場合、**Details** を選択して、変更が適用された OpenTelemetry のウェブサイトのステージングバージョンを開いてください。 これがレビュアーが変更を確認する方法です。
+
+他のチェックも同様に失敗している可能性があります。[全ての PR チェック](../pr-checks) を参照してください。
+
+### 内容の問題を自動的に修正する {#fix-content-issues-automatically}
+
+リポジトリに変更を提出するまえに、以下のコマンドを実行して (i) 報告された問題の対応し、(ii) スクリプトによって変更されたファイルのコミットをしてください。
+
+```sh
+npm run test-and-fix
+```
+
+ファイルに対して、テストとすべての問題の修正を分割して実行するには、以下を実行してください。
+
+```sh
+npm run test    # ファイルを更新せずにチェックのみを実行します
+npm run fix:all # ファイルを更新する場合があります。
+```
+
+利用可能な NPM スクリプトのリストを表示するには、`npm run` を実行してください。
+プルリクエストのチェックとエラーの自動修正の詳細は、[PR checks](../pr-checks) を参照してください。
+
+### 変更をローカルでプレビューする {#preview-locally}
+
+変更のプッシュまたはプルリクエストの公開の前に、ローカルでプレビューしてください。
+プレビューはビルドエラーとマークダウンのフォーマットの問題を検出できます。
+
+Hugo をローカルでビルドと提供するには、以下のコマンドを実行してください。
+
+```shell
+npm run serve
+```
+
+Web ブラウザで `http://localhost:1313` に移動してローカルプレビューを確認してください。
+Hugo は変更を監視し、必要に応じてサイトをリビルドします。
+
+ローカルの Hugo インスタンスを停止するには、ターミナルにもどって `Ctrl+C` を入力するか、ターミナルを閉じてください。
+
+### サイトデプロイと PR プレビュー {#site-deploys-and-pr-previews}
+
+PR を提出したら、Netlify は [deploy preview][] を作成し、変更をレビューできます。
+PR がマージされると、Netlify はプロダクションサーバーに更新されたサイトを本番サーバーにデプロイします。
+
+> **Note**: PR プレビューには ドラフトページ が含まれますが、本番ビルドには含まれません。
+
+デプロイログなどを確認するには、プロジェクトの [dashboard][] を確認してください。
+Netlify ログインが必要です。
+
+### PR ガイドライン {#pr-guidelines}
+
+PR がマージされる前に、レビューと編集を数回繰り返すことがあります。
+このプロセスをできるだけ簡単にするために、以下のガイドラインに従ってください。
+
+- もしあなたの PR が簡単な修正でない場合は、**フォークから作業**してください。 リポジトリの上部にある [Fork](https://github.com/open-telemetry/opentelemetry.io/fork) ボタンをクリックし、フォークをローカルにクローンしてください。準備ができたら、アップストリームリポジトリに PR を作成してください。
+- あたなたのフォークの **`main` ブランチから作業をせずに**、PR 専用のブランチを作成してください。
+- メンテナーが[あなたのプルリクエストに変更を加えられること](https://docs.github.com/ja/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)を確認してください。
```

Comment:
Oh! Good idea to link to the Japanese translation of the GitHub docs page. Let me see if I can fix this in the `en` version using a `param`.

Reply:
@chalin 

Thank you for your advice! However, I might not be able to understand your intention. Which is correct adding  `params.link_name` or `params.links.linkname` in `hugo.yaml`?

I created sample commit of it.

https://github.com/open-telemetry/opentelemetry.io/pull/6197/commits/6114eb19d71cbc769a77c9ee1135afa8da22e809

Please check it and advice more information.
Thank you.

Reply:
There wasn't anything for you to do based on my original comment. Sorry for the confusion. I only wanted to leave a note (mostly to myself) to look into an i18n feature.

While 6114eb19d71cbc769a77c9ee1135afa8da22e809 is a good change, it isn't one that I'd want implemented in this PR. Please revert it. Thank you.

Reply:
Oh, ok. I understand. I was mistaken.

I've made a revert commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6197/commits/1987b4d9ed6e1e3caa4bbb276d265f003145eba8